mobile/lib/widgets/custom_widgets.dart

```diff
@@ -495,7 +495,7 @@ class AppSafeArea extends StatelessWidget {
           vertical: verticalPadding ?? 0,
           horizontal: horizontalPadding ?? 0,
         ),
-        child: widget,
+        child: child,
```

Comment:
Could it be possible to just use "child" without any assignment sin ce the value's name is the same as the key's name?

Reply:
Thanks @Baalmart for this. We can achieve it by using positional parameters. The motivation to use chid is to follow flutter's convention. It uses `child` and `children` named parameters to represent child/children widget(s). 

In this case `AppSafeArea` is a parent widget that `"extends"` flutter's inbuilt  SafeArea widget to include vertical and horizontal padding.

cc @Mnoble-19 @BenjaminSsempala